### PR TITLE
use the latest published version of the helm chart

### PR DESCRIPTION
**Pull request by djeebus**
Currently this requires that two repositories are checked out locally, and in a specific place. something like:

```
/c7n-helm-charts   = github.com/cloud-custodian/helm-charts
/cloud-custodian   = github.com/cloud-custodian/cloud-custodian
```

Specifically, they need to be checked out next to each other, and the helm charts repo MUST be called `c7n-helm-charts`. This makes the local development process brittle and hard to set up for a first time experience.

If we make the assumption that iterating on the container generally won't mean iterating on the helm chart as well, we can simplify this by using the released version of the chart and a static set of values that are known to work with skaffold. 

#7920 was accidentally closed, and I couldn't reopen it. Reopened here.

### CODE REVIEW

**Comment by djeebus** [comment]
Found the `repo` field, which let us point this at the latest chart, rather than a specific one.

**Comment by djeebus** [comment]
@sontek @kapilt

File: tools/c7n_kube/.skaffold/values.yaml [hunk 1]
```diff
@@ -1,24 +1,48 @@
...
+              - metadata.labels."app.kubernetes.io/managed-by": absent
+
+webhook:
+  caBundle: blahblahblah
+
+  namespaceSelector:
+    matchExpressions:
+      - key: c7n-monitor
+        operator: NotIn
+        values: [ "false" ]
```
Comment (sontek) [inline tools/c7n_kube/.skaffold/values.yaml:41]: Do we need this namespace selector or the caBundle on this?

**Comment by djeebus** [comment]
whoops, I can remove the caBundle; the namespace selector is useful for avoiding running policies on the operator itself

File: tools/c7n_kube/.skaffold/values.yaml (hunk 1 above)
Comment (djeebus) [reply tools/c7n_kube/.skaffold/values.yaml:41]: whoops, I can remove the caBundle; the namespace selector is useful for avoiding running policies on the operator itself

**Comment by sontek** [comment]
@djeebus Thats not the namespace we run it in though, is it?  Should we update that to `c7n-system`?

**Comment by djeebus** [comment]
That's a label, indicates not to monitor any namespaces that have the label `c7n-monitor: "false"`. Just for testing.

**Comment by sontek** [comment]
oh, sorry I misread that.   👍 looks good to me

**Review by kapilt — APPROVED** [review]
lgtm

**Pull request merged**